### PR TITLE
Update Firefox Desktop Attribution docs

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Updates Firefox stub attribution documentation to better reflect the state of the current system.

## Issue / Bugzilla link

N/A

## Testing

https://bedrock.readthedocs.io/en/latest/install.html#documentation

### CODE REVIEW

**Review by craigcook — CHANGES_REQUESTED** [review]
Looks good and makes sense. Of course I'm no expert so can't verify accuracy, but I was able to follow the explanation. Just one term I was confused by, and a suggestion for moving the flowchart.

File: docs/firefox-desktop-attribution.rst [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+.. _Do Not Track (DNT): https://support.mozilla.org/kb/how-do-i-turn-do-not-track-feature
+.. _privacy policy: https://www.mozilla.org/privacy/websites/
+
+How does attribution work?
+--------------------------
+
+See the `Application Logic Flow Chart`_ for a visual representation of the steps below.
+
+#. A user visits a page on www.mozilla.org. On page load, a `JavaScript
+   function`_ collects referral and analytics data about frp, where their visit
```
[the remark this answers is not in the document]
Comment (craigcook) [reply docs/firefox-desktop-attribution.rst:44]: I just realized this is probably supposed to be "from" and you were one key to the right 😁

File: docs/firefox-desktop-attribution.rst [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+   service then stores both the attribution data (including the Google Analytics
+   client ID) and the download token in Mozilla's private server logs.
+#. The service then passes the download token and attribution data (excluding the
+   GA client ID) into the installer being served to the user.
+#. Once the user installs Firefox, the data that was passed to the installer is
+   then stored in the users' Telemetry profile.
+#. During analysis, the download token can be used to join Telemetry data
+   with the corresponding GA data in the server logs.
+
+.. _Application Logic Flow Chart: https://www.figma.com/file/q5mJpicWBpzAYuQ3fV00ix/Firefox-Stub-Attribution-Flow?node-id=0%3A1&t=EFe91WQzQ7cXHSiB-1
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply docs/firefox-desktop-attribution.rst:73]: Will add a note that this link is mozilla access only. I think that's fine, seeing as this is something mostly for our own team to reference.

**Comment by alexgibson** [comment]
Thanks @craigcook! Updated

**Review by craigcook — APPROVED** [review]
📜

**Pull request merged**